examples/demo_launcher/lib/main.dart

```diff
@@ -177,7 +177,7 @@ class DemoList extends Component {
   Widget build() {
     return new ScrollableList<SkyDemo>(
       items: demos,
-      itemHeight: kCardHeight,
+      itemSize: kCardHeight,
```

Comment:
Thank  you for "extent"!  I've used that term [for this kind of thing before](http://docs.oracle.com/javase/7/docs/api/javax/swing/BoundedRangeModel.html) but could not recall it when it came time to rename itemHeight.  Perhaps we should rename scrollOffset (the double) scrollExtent?